docs/reference/setup/install/zip-windows-start.asciidoc

```diff
@@ -7,5 +7,9 @@ Elasticsearch can be started from the command line as follows:
 .\bin\elasticsearch.bat
 --------------------------------------------
 
+If you have configured secure settings and password-protected the {es}
+keystore, you will be prompted to enter the keystore's password. See
+{ref}/secure-settings.html[secure settings] for more details.
```

Comment:
```suggestion
<<secure-settings>> for more details.
```

Reply:
I know our docs are divided into books, but what determines that boundary? Is it the top-level sections in the sidebar?

Reply:
Each of the items on https://www.elastic.co/guide/index.html is considered a book.  The elasticsearch repo is a little tricky, since there are multiple "books" that are generated from that repo, so you can't always assume that another page in the same repo is in the same book.  The concepts of "books" might go away, however, so it's not worth worrying too much about.